internal/kibana/reconciler.go

```diff
@@ -34,6 +35,8 @@ const (
 	expectedCLOName              = "instance"
 	expectedCLOKibana            = "kibana"
 	expectedCLONamespace         = "openshift-logging"
+	oauthProxyImageName          = "oauth-proxy"
+	oauthProxyNamespace          = "openshift"
```

Comment:
This seems to be only used to initialize the `client.ObjectKey` used to find the ImageStream of the proxy. Does it make sense to initialize a global variable for the ObjectKey instead of using these two constants?

Reply:
I will use constants for consistency with our codebase.